yarn.lock

```diff
@@ -14404,7 +14404,7 @@ ts-loader@^4.1.0:
     micromatch "^3.1.4"
     semver "^5.0.1"
 
-tslib@^1.10.0, tslib@^1.8.0, tslib@^1.8.1, tslib@^1.9.0:
+tslib@^1.10.0, tslib@^1.7.1, tslib@^1.8.0, tslib@^1.8.1, tslib@^1.9.0:
```

Comment:
Per my other comment, we shouldn't see any changes in this file.

Reply:
weird!

Reply:
Well, this change is caused by the tslib `^1.7.1` usage above. When you change it to `^1.10.0`, this file should revert to having no changes. :)

Reply:
I fixed this.